cloud/endagaweb/templates/dashboard/subscriber_detail/adjust_credit.html

```diff
@@ -59,6 +61,9 @@
     {% endif %}
   </div>
 </div>
+{% else %}
+{% include 'dashboard/permission_denied.html' %}
```

Comment:
PermissionRequiredMixin is added in the view and permission_required is for view_*,
We have added this to render permission_denied.html with context,
if we add in view the template will not have any context and user will be kicked out from the layout, just like the current 404 view.

if that is what we looking for we can update the current implementation on that basis.